latch_cli/services/init/init.py

```diff
@@ -141,28 +151,54 @@ def _gen_example_conda(pkg_root: Path):
     shutil.copy(conda_env_src, conda_env_dest)
 
 
+def _gen_example_docker(pkg_root: Path):
+    import boto3
+    from botocore import UNSIGNED
+    from botocore.config import Config
+
+    pkg_root = pkg_root.resolve()
+    source_path = Path(__file__).parent / "assemble_and_sort"
+
+    _get_boilerplate(pkg_root, source_path)
+    _get_example_reference(pkg_root)
+
+    source_docker_path = Path(__file__).parent / "example_docker"
+    shutil.copy(source_docker_path / "assemble.py", pkg_root / "wf/assemble.py")
+
+    (pkg_root / ".env").unlink()
+
+    print()
+
+
 option_map = {
     "Empty workflow": _gen_template,
     "Subprocess Example": _gen_assemble_and_sort,
     "R Example": _gen_example_r,
     "Conda Example": _gen_example_conda,
+    "Docker Example": _gen_example_docker,
 }
 
 
 template_flag_to_option = {
     "empty": "Empty workflow",
+    "docker": "Docker Example",
     "subprocess": "Subprocess Example",
     "r": "R Example",
     "conda": "Conda Example",
 }
 
 
+class BaseImageOptions(Enum):
```

Comment:
add a default option here so that the default value in the `init` function makes sense, and so that people can specify `--base-image default` if they want to

Reply:
would be easier to do if `default` were named `base` but up to u